### DISCUSSION TITLE
MeetUp?

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#2662] [Polls]
It's lonely behind a terminal, I get it. Besides: it's fun meeting people who share your passions! Brainstorming, collaborating, and debugging can be a lot more fun in person.

**Comment by banesullivan** [comment]
Fantastic idea!! I know @akaszynski, @tkoyama010, and I will be at SciPy this July - perhaps that would be a good time and place for a physical meet up for any others attending?

Otherwise, I'd love to coordinate some sort of larger, virtual meetup/hackathon

**Reply by tkoyama010** [reply]
We may be able to organize Meetings like the [simpeg project](https://simpeg.xyz/).
https://www.youtube.com/c/GeosciXyz/videos
@adam-grant-hendry 
If you want to do it online, I can help.
This meeting might be helpful to free up time for development and tutorial creation.

Also, we will have a [sprint at Scipy2022](https://www.scipy2022.scipy.org/sprints).
Please consider joining us there.

_Originally posted by @tkoyama010 in https://github.com/pyvista/pyvista/discussions/2661#discussioncomment-2765036_
(Since the threads were duplicated I locked the other thread and merged the conversation here.)

**Reply by adam-grant-hendry** [reply]
Thanks @banesullivan and @tkoyama010 ! I wish I could go to SciPy, esp. to meet you guys! I'll be at a family reunion then, unfortunately.

@tkoyama010 Let's set up an online thing too! I do also want to set up time for in-person work in Irvine, CA, or elsewhere if anyone is also interested in that. Regardless, I find it personally beneficial to get a face to a name so when I'm commenting/talking online I have greater context of whom I'm working with...if that makes sense 😄 .

**Comment by tkoyama010** [comment]
It may be a good idea to set up a Discord as a community chat. You can allow other users to talk to you if you use the voice channel.
https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord
![Video_Chat_-_Focused](https://support.discord.com/hc/article_attachments/360060868111/Video_Chat_-_Focused.png)

**Reply by adam-grant-hendry** [reply]
Love this idea too!

**Reply by tkoyama010** [reply]
@adam-grant-hendry See https://github.com/pyvista/pyvista/discussions/2749
